scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -188,16 +195,30 @@ task ExtractTask {
                 ~{true='--emit-pls' false='' emit_pls} \
                 ${FILTERING_ARGS}
 
-        du -b ~{output_file} | cut -f1 > vcf_bytes.txt
-        du -b ~{output_file}.tbi | cut -f1 > vcf_index_bytes.txt
+        INTERVAL_NUMBER=$(echo ~{output_file} | grep -oEi '[0-9]+\.vcf\.gz' | cut -d'.' -f1)
```

Comment:
This is complicated and also breaks layers of abstraction by assuming a specific filename format. If you need this, I would add a new input parameter like `interval_index`